Tests/NIOTests/EventLoopTest.swift

```diff
@@ -130,13 +134,17 @@ public class EventLoopTest : XCTestCase {
         }
 
         let expect = expectation(description: "Is cancelling RepatedTask")
-        var repeatedTask: RepeatedTask?
-        repeatedTask = eventLoopGroup.next().scheduleRepeatedTask(initialDelay: initialDelay, delay: delay) { () -> Void in
+        let group = DispatchGroup()
+        group.enter()
+        let repeatedTask = eventLoopGroup.next().scheduleRepeatedTask(initialDelay: initialDelay, delay: delay) { () -> Void in
             DispatchQueue(label: "nio.repeatedtask.test").async {
```

Comment:
This test can avoid using this `DispatchQueue`, because the `.notify` will use a different thread anyway.